CriteoPublisherSdk/Sources/CR_BidManager.h

```diff
@@ -50,6 +50,7 @@
 @property(nonatomic, readonly) CR_Config *config;
 @property(nonatomic, strong) CR_DataProtectionConsent *consent;
 @property(nonatomic, strong) CR_ThreadManager *threadManager;
+@property(nonatomic, strong) NSNumber *childDirectedTreatment;
```

Comment:
nullable

Reply:
While technically correct, one cannot specify the nullability of only one variable inside a class. All the variables need to have nullability indicators. I believe this type of change is outside the scope of this ticket.
![can't have only one nullable](https://user-images.githubusercontent.com/111067505/188121982-40fba336-7aab-47d0-b702-3f2c15a105ab.png)